code/implementation/host-stats/src/main/java/io/cattle/platform/host/stats/api/StatsLinkHandler.java

```diff
@@ -0,0 +1,93 @@
+package io.cattle.platform.host.stats.api;
+
+import io.cattle.platform.api.link.LinkHandler;
+import io.cattle.platform.archaius.util.ArchaiusUtil;
+import io.cattle.platform.core.constants.HostConstants;
+import io.cattle.platform.core.constants.InstanceConstants;
+import io.cattle.platform.core.model.Host;
+import io.cattle.platform.core.model.Instance;
+import io.cattle.platform.docker.util.DockerUtils;
+import io.cattle.platform.host.model.HostApiAccess;
+import io.cattle.platform.host.service.HostApiService;
+import io.cattle.platform.host.stats.utils.StatsConstants;
+import io.cattle.platform.object.ObjectManager;
+import io.github.ibuildthecloud.gdapi.request.ApiRequest;
+
+import java.io.IOException;
+import java.util.Collections;
+
+import javax.inject.Inject;
+
+import com.netflix.config.DynamicStringProperty;
+
+public class StatsLinkHandler implements LinkHandler {
```

Comment:
Can you please add a `@Deprecated` annotation stating that this will be removed once the UI switches to using the new stats links?